requirements.txt

```diff
@@ -8,7 +8,7 @@ django-flat-json-widget~=0.1.2
 # TODO: change this when next version of openwisp_users is released
 openwisp-users @ https://github.com/openwisp/openwisp-users/tarball/master
 openwisp-utils[rest]~=0.7.1
-openwisp-notifications~=0.3
+openwisp-notifications @ https://github.com/niteshsinha17/openwisp-notifications/tarball/issues/164-introduce-mechanism-to-clear-cache
```

Comment:
reminder: before merging let's switch this back to master once that PR has been merged.

Reply:
let's switch this to master